packages/sdk-codegen-scripts/src/convert.spec.ts

```diff
@@ -24,7 +24,7 @@
 
  */
 
-import { TestConfig } from '../../test-utils/src/testUtils'
+import { TestConfig } from '@looker/sdk-test-utils'
```

Comment:
Deep imports will break the dynamic package import for Jest and aren't really "proper form" anyway